src/app/presentation/feedback-slide/feedback-slide.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="slide-size feedback-slide-background">
+
+<div class="content">
+  <div class="flex-container">
+    <img class="feedback-slide-header feedback-slide-logo" src="/assets/images/angular-notice.png">
+    <span class="feedback-slide-header feedback-slide-title">NOTICE</span>
+  </div>
+
+  <br>
+
+  <div class="feedback-slide-description">
+      Chase imaginary bugs love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater) leave dead animals as gifts, so sniff other cat's butt and hang jaw half open thereafter has closed eyes but still sees you or thug cat run in circles.
+  </div>
+
+  <div class="feedback-button-title">
+    FEEDBACK BUTTON
+  </div>
+
+  <div class="feedback-button-ripple-1">
```

Comment:
can this be a for loop? 

```html
<div *ngFor="#number of numberCollection" [ngClass]="'feedback-button-ripple-'+number">
    <div class="fadeOutUp"></div>
</div>
```

where `numberCollection` is a collection of numbers (ex: [1,2,...24]) from the component... unless there is a better way to iterate from 1 to 24? I am not 100% on this.